Fixed code inconsistencies added method-chaining possibilities and im…

mikevandiepen opened the pull request:
What I changed:
* Readability of the code
* Implemented PSR12 standards 
* Added docblocks in the ReviewFactory
* Implemented method chaining on "setters"
* Fixed code inconsistencies.

I have also noticed that there are a lot of not-used setters? maybe get rid of them?

Commit:
Fixed code inconsistencies added method-chaining possibilities and implemented PSR12.

JKetelaar commented:
Hi, great change, thank you!

I am not a fan of the indenting of variables etc (makes me think of WordPress brrrr), but I will change that myself. 

Could you fix the merge conflicts, so I can continue working on it?

Commit:
Fixed merge conflict and updated some code of other maintainer

mikevandiepen commented:
Fixed merge conflict, also it might be about time to set some baselines for code quality? I saw that some methods without any docblock got added.

Also I removed an if statement out of a switch statement.. Which is a terrible code practice!

I also fixed the indentation for you, I'm also not a big fan of it but it improves readability by a lot.

JKetelaar commented:
I have merged this pull request but I will fully revamp this, as this is not using the Symfony coding standards, which is what I am using. I will be adding the coding standards to the repository.

Thanks again for your contribution, this is greatly appreciated!
